src/Uno.UI/UI/Xaml/Controls/ComboBox/ComboBox.cs

```diff
@@ -779,6 +779,25 @@ Point getChildLocation()
 				}
 
 				var comboRect = combo.GetAbsoluteBoundsRect();
+
+#if __IOS__
+				// We might get to a situation where the calculations performed during
+				// .GetAbsoluteBoundsRect() did not return the correct value
+				// ex: When the Parent View is not found.This can happen when MultiFrame
+				// Navigation is used to display a Modal Page.
+				// With this, we ensure we use the ComboBox's Location when displaying
+				// the Popup with the options.
+				if (comboRect.X < combo.Frame.X)
+				{
+					comboRect.X = combo.Frame.X;
+				}
+
+				if (comboRect.Y < combo.Frame.Y)
+				{
+					comboRect.Y = combo.Frame.Y;
+				}
```

Comment:
Did you find a way to re-create the issue in a runtime test? I'm not sure if it will work in a simulator (we're running on tablet for this test).